### PR TITLE
Feature(api-auth): add logto get access token for Fastify

### PR DESCRIPTION
**Pull request by SamSalvatico**
### Description

Get access token on the Fastify side

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [x] **New feature**
- [ ] **Dev change**

### CODE REVIEW

File: apps/payments-api/openapi-definition.yml [hunk 1]
```diff
@@ -1,2542 +0,0 @@
-openapi: 3.0.3
```
Comment (peschina) [inline apps/payments-api/openapi-definition.yml:1]: I see the whole schema deleted here, is this expected?

**Review by peschina — DISMISSED** [review]
LGTM, one small comment

File: apps/payments-api/openapi-definition.yml (hunk 1 above)
Comment (SamSalvatico) [reply apps/payments-api/openapi-definition.yml]: can't understand why, fixed, thank you Sara

**Review by peschina — APPROVED** [review]
LGTM

**Pull request merged**